Xamarin.Forms.Core/BindableLayout.cs

```diff
@@ -0,0 +1,262 @@
+﻿using System.Collections;
+using System.Collections.Specialized;
+
+namespace Xamarin.Forms.Core
+{
+	public static class BindableLayout
+	{
+		public static readonly BindableProperty ItemsSourceProperty =
+			BindableProperty.CreateAttached("ItemsSource", typeof(IEnumerable), typeof(Layout<View>), default(IList),
+				propertyChanged: (b, o, n) => { GetController(b).ItemsSource = (IEnumerable)n; });
+
+		public static readonly BindableProperty ItemTemplateProperty =
+			BindableProperty.CreateAttached("ItemTemplate", typeof(DataTemplate), typeof(Layout<View>), default(DataTemplate),
+				propertyChanged: (b, o, n) => { GetController(b).ItemTemplate = (DataTemplate)n; });
+
+		public static readonly BindableProperty ItemTemplateSelectorProperty =
+			BindableProperty.CreateAttached("ItemTemplateSelector", typeof(DataTemplateSelector), typeof(Layout<View>), default(DataTemplateSelector),
+				propertyChanged: (b, o, n) => { GetController(b).ItemTemplateSelector = (DataTemplateSelector)n; });
+
+		static readonly BindableProperty ControllerProperty =
```

Comment:
Should this be `BindableLayoutControllerProperty` ?

Reply:
Done!